mesop/dataclass_utils/dataclass_utils.py

```diff
@@ -103,6 +103,13 @@ def _recursive_update_dataclass_from_json_obj(instance: Any, json_dict: Any):
       else:
         # For other types, set the value directly.
         setattr(instance, key, value)
+    else:
```

Comment:
It won't let me comment on line 88, so commenting here.

It's not related to this PR.  But what happens in the nested list case?

Reply:
added a test case to cover nested list - it works.